src/detect-transform-strip-whitespace.c

```diff
@@ -72,6 +75,26 @@ static int DetectTransformStripWhitespaceSetup (DetectEngineCtx *de_ctx, Signatu
     SCReturnInt(r);
 }
 
+/*
+ *  \brief Validate content string to see if it's compatible with this transform
+ *  \param string String to check for compatibility
+ *  \param len Number of string bytes to check
+ *  \param options Ignored
+ *  \retval false If the string contains spaces
+ *  \retval true Otherwise.
+ */
+static bool TransformStripWhitespaceValidate(const uint8_t *string, uint16_t len, void *options)
+{
+    if (string && len) {
+        while (len-- && *string) {
```

Comment:
the `*string` condition would make the loop bail on `\0` right?

`content:"|00 20|";` would not be handled properly then.

Reply:
Good catch; will fix.